metabase/metabase
# Simplify additional options in set up flow db configuration

Epic: https://github.com/metabase/metabase/issues/19057

1. Removed IP address block
2. Added `info` setting type to add fields like IP address by the backend
3. Added `section` setting type for hiding advanced options, to be used by the backend with `visible-if`.

How to test:
1. Open https://github.com/metabase/metabase/blob/40bb84ffec3b4ea410d9c8f303204ac10ea63442/frontend/src/metabase/entities/databases/forms.js#L400
2. Add sample settings:
```
 {
  name: "info_example",
  type: "info",
  placeholder: "Sample content",
},
{
  name: "section_example",
  type: "section",
},
```
3. Open `/admin/databases/create`. Verify that new fields have been displayed correctly.

<img width="672" alt="Screen Shot 2021-11-22 at 6 12 50 pm" src="https://user-images.githubusercontent.com/8542534/142886371-70483d48-acbd-472d-89c0-e87e0be06c78.png">

There are no new e2e tests for this as the backend part is not implemented yet.
